features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+
+    @ui
+    Scenario: When a page is not found a custom 404 error page is displayed
+        Given the user clicks on a link which is not available
+        Then they see a 404 Page Not Found message
+        And they see a custom design with Squatchy sitting in the middle of the 0
+        And a message:
+            """Please excuse the mess—We are in the process of making our documentation even better.
+    If you cannot find the information you're looking for, let our Success team know, and they will be more than happy to provide!"""
+
+    @ui
+    Scenario: The logo at the top of the side navbar takes you to the "https://docs.saasquatch.com/" page
+        Given the user is on any page of the docs portal
+        When they click the SaaSquatch logo at the top of the navigation bar
+        Then the navigation bar has every dropdown closed
+        And shows the 6 main menu items
+        And the central section of the page shows the "help center" search page
+        And the url is "https://docs.saasquatch.com/"
+
+    Scenario: There is a search box at the top of the side navbar (more specs available in a separate file for Search)
+        Given the user is on the SaaSquatch docs page at any location
+        Then a search input field is shown in the sidebar
+        And the default state for the search input field is empty
+        And the search is working expected
+
+    Scenario: In the side navbar there are different menu item types
+        Given the user is on the docs portal
+        Then they see the following side navbar item types
+
+            | types                                                    |
+            | Core Category Section with navigation arrow              |
+            | Core Category Section with icon only                     |
+            | Navigation item with an arrow down                       |
+            | Article item                                             |
+            | Core category item at the top of the navbar with an icon |
+
+    Scenario: The Core Category Section with navigation arrow takes you to another level of navigation items and opens up a page
+        Given the side navbar is in the starting state
+        When the user clicks on any part of a " Core Category Section with navigation arrow"
+        Then they see that in the nav bar a "Core category item at the top of the navbar with an icon" shows up
+        And the sub-category items are there under it
+        And the home page opens up for the selected core category
+
+    Scenario: Core Category Section with icon only
+        Given the side navbar is in the starting state
+        Then SaaSquatch Product News are displayed as the last menu item without an arrow
+        When the user clicks on this item
+        Then a home page for Product News opens up
+        And there are no sub-categories for this item type
+
+    Scenario: The navigation item with an arrow down opens a dropdown menu
```

Comment:
Just giving a suggestion for a scenario title, try to apply this to other scenario titles
```suggestion
    Scenario: Sidebar drop down elements display sub articles when clicked
```